0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -8,7 +8,7 @@ draft: 7
 
 Interledger provides for secure payments across multiple assets on different ledgers. The architecture consists of a conceptual model for interledger payments, a mechanism for securing payments, and a suite of protocols that implement this design.
 
-The [Interledger Protocol (ILP)](#interledger-layer) is the core of the Interledger protocol suite. Colloquially, the whole Interledger stack is sometimes referred to as "ILP". Technically, however, the Interledger Protocol is only one layer in the stack.
+The Interledger Protocol (ILP) is the core of the Interledger protocol suite. Colloquially, the whole Interledger stack is sometimes referred to as "ILP". Technically, however, the Interledger Protocol is only one layer in the stack.
```

Comment:
This link no longer exists because the content was rewritten in https://github.com/interledger/rfcs/commit/a0b7cf13d9c6df28dda907973635396f421f692a#diff-789016c97bee3d0394e54f3d37ae423f93833b26c88bfa6672250d4d02c0bd17L81